src/actions/projects.js

```diff
@@ -0,0 +1,156 @@
+import axios from "axios";
+
+const baseUrl = "http://localhost:4000";
+
+export const PROJECTS_FETCHED = "PROJECTS_FETCHED";
+
+export function projects(projects) {
+  return {
+    type: PROJECTS_FETCHED,
+    payload: projects
+  };
+}
+
+export function loadProjects() {
+  return async function(dispatch) {
+    try {
+      const response = await axios.get(`${baseUrl}/project`);
+      const { data } = response;
+
+      const action = projects(data);
+      dispatch(action);
+    } catch (error) {
+      throw error;
+    }
+  };
+}
+export const ONE_PROJECT_SELECTED = "ONE_PROJECT_SELECTED";
```

Comment:
define all your actionTypes on the top together so when you open the file you can quickly see what actions it manages